parts/kubernetesmasteraddons-tiller-deployment.yaml

```diff
@@ -5,13 +5,15 @@ metadata:
   namespace: kube-system
   labels:
     kubernetes.io/cluster-service: "true"
```

Comment:
@lachie83 do you approve of retaining the `kubernetes.io/cluster-service: "true"` label with the introduction of `addonmanager.kubernetes.io/mode=EnsureExists`?